homeassistant/helpers/__init__.py

```diff
@@ -98,14 +98,17 @@ def config_per_platform(config, domain, logger):
 
     while config_key in config:
         platform_config = config[config_key]
+        if type(platform_config) is not list:
+            platform_config = [platform_config]
 
-        platform_type = platform_config.get(CONF_PLATFORM)
+        for item in platform_config:
+            platform_type = item.get(CONF_PLATFORM)
 
-        if platform_type is None:
-            logger.warning('No platform specified for %s', config_key)
-            break
+            if platform_type is None:
+                logger.warning('No platform specified for %s', config_key)
+                break
```

Comment:
Let's change this to `continue`, so that if you specify 3 platforms and forget `CONF_PLATFORM` on the second one, we will still load the 3rd one.